### ISSUE TITLE
Tiny Audio M17+

### ISSUE THREAD

**Issue by atchoo78** [cweiske/frontier-silicon-firmwares#34]
I have a Tiny Audio M7+ (Pinell?) with firmware:

```
ir-mmi-FS2026-0500-0213.2.6.20c8.EX63098-1MA15
```

**Comment by cweiske** [comment]
Thanks!

**Comment by atchoo78** [comment]
The manufacturer is actually "TT Micro" (Norwegian). By the way, I just discovered that the M7+ uses port **8080** for XML files related to transport controls (play, pause, stop etc) and UPnP/SSDP discovery, which enables the "hidden" DMR input mode for streaming "local" files. 

I don’t know if it’s of any significance though, since it’s kind of hard to tell if it’s made by Frontier (i.e. part of the firmware), or if it was added by the manufacturer specifically for that device/radio. I just thought I’d mention it, just in case.

Anyway, these are the URLs I’ve discovered so far: 

```
http://192.168.x.x:8080/dd.xml
http://192.168.x.x:8080/ConnectionManager/scpd.xml
http://192.168.x.x:8080/AVTransport/scpd.xml
http://192.168.x.x:8080/RenderingControl/scpd.xml

```
